spec/resources/productType2.json

```diff
@@ -12,6 +12,78 @@
         "name": "text"
       },
       "attributeConstraint": "SameForAll"
+    },
+    {
+      "name": "sameForAll-constraint-attr",
```

Comment:
These 4 new attributes are not used anywhere in the project:
 - no-constraint-attr
 - unique-constraint-attr
 - sameForAll-constraint-attr
 - combination-unique-constraint-attr

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/42/commits/1b3f0c02106016f8c8eb4668311d33aaf701e177

Reply:
I removed the whole file: https://github.com/commercetools/commercetools-node-variant-reassignment/pull/42/commits/523a653304d6d0bf5167f878a905dbfbd91a03db